gph/bindings/ripser_bindings.cpp

```diff
@@ -15,85 +15,84 @@
 namespace py = pybind11;
 
 #if defined USE_COEFFICIENTS
-PYBIND11_MODULE(gph_ripser_coeff, m)
-{
+PYBIND11_MODULE(gph_ripser_coeff, m) {
 #else
-PYBIND11_MODULE(gph_ripser, m)
-{
+PYBIND11_MODULE(gph_ripser, m) {
 #endif
 
-    using namespace pybind11::literals;
-    m.doc() = "Ripser python interface";
+  using namespace pybind11::literals;
+  m.doc() = "Ripser python interface";
 
-    // Because `ripser` could have two different modules after compilation
-    // It's necessary to add `py::module_local()` to prevent following issue:
-    // ImportError: generic_type: type "ripserResults" is already registered!
-    // When same python module imports gtda_ripser and gtda_ripser_coeff
-    py::class_<ripserResults>(m, "ripserResults", py::module_local())
-        .def_readwrite("births_and_deaths_by_dim",
-                       &ripserResults::births_and_deaths_by_dim)
-        .def_readwrite("num_edges", &ripserResults::num_edges);
+  // Because `ripser` could have two different modules after compilation
+  // It's necessary to add `py::module_local()` to prevent following issue:
+  // ImportError: generic_type: type "ripserResults" is already registered!
+  // When same python module imports gtda_ripser and gtda_ripser_coeff
```

Comment:
Are these comments still relevant?  Because they refer to `gtda_ripser` and `gtda_ripser_coeff`

Reply:
They are still relevant, but as you pointed out they are not up-to-date